sdk/compiler/daml-extension/BUILD.bazel

```diff
@@ -47,7 +47,10 @@ sh_test(
 # exists", so at this point I'm completely out of trust.
 genrule(
     name = "node_deps_cache",
-    srcs = ["@daml_extension_deps//:node_modules"],
+    srcs = [
+        "@daml_extension_deps//:node_modules",
+        "//compiler/daml-extension:yarn.lock",
```

Comment:
We've added the yarn.lock here, but we continue to depend on it in `genrule("vsix"`, where (I think) we overwrite it when unpacking this tarball. We should either stop tarballing it here or stop depending on it there (unless I'm missing something?).